custom-monitor/thousandeyes/monitor.py

```diff
@@ -207,7 +208,7 @@ def query_latest_data (username, authtoken, accountname, testname, window_second
 
         for agentdata in testDetails['agents']:
             testdata = teApi.getTestPageloadData(test['testId'])
-            if testdata :
+            if testdata and 'web' in testdata.keys() :
```

Comment:
@brownkw - should add this check for http and net as well.

Reply:
@hashlock-te Not sure how I missed that. Those checks should be present now.